README.md

```diff
@@ -2,7 +2,7 @@
 
 # Portfolio
 
-> A test Portfolio with Figma
+> Victoria's Portfolio. Please visit my live portfolio page using this [link](https://victoria129.github.io/). 
```

Comment:
[OPTIONAL]  Great work on this. I suggest you try and capture attention of the reader by using syntax like
 ... 
## [Live Demo](https://victoria129.github.io/)
This will catch your user's attention quicker and save their time.